PencilChiselCode/Source/GameStates/MenuState.cs

```diff
@@ -5,7 +5,7 @@
 using MonoGame.Extended.Screens;
 using MonoGame.Extended.Screens.Transitions;
 
-namespace PencilChiselCode.Source;
+namespace PencilChiselCode.Source.GameStates;
```

Comment:
Why is this changed?

Reply:
![image](https://user-images.githubusercontent.com/26803198/201475893-bfc19497-5b71-4ef6-b947-1ceccd1b2775.png)